guides/common/assembly_administering-hosts.adoc

```diff
@@ -18,8 +18,14 @@ include::modules/proc_creating-a-host-group.adoc[leveloffset=+1]
 
 include::modules/proc_creating-a-host-group-for-each-lifecycle-environment.adoc[leveloffset=+1]
 
+include::modules/proc_adding-a-host-to-a-host-group.adoc[leveloffset=+1]
+
 include::modules/proc_changing-the-host-group-of-a-host.adoc[leveloffset=+1]
 
+ifdef::katello,satellite,orcharhino[]
+include::modules/proc_adding-a-host-to-a-host-collections.adoc[leveloffset=+1]
```

Comment:
```suggestion
include::modules/proc_adding-a-host-to-a-host-collection.adoc[leveloffset=+1]
```

I tried to rename the file, but there already is a file of this name. Pls, clean up.